docs/domains/KnowledgeArtifactManagement/services/TerminologyService/openapi.yml

```
@@ -0,0 +1,540 @@
+    get:
+      tags:
+        - Fhir
+      summary: "Returns the CapabilityStatement describing the functionalities\r\nand conformance requirements of the FHIR Terminology server."
+      responses:
+        '200':
+          description: OK
+          content:
+            application/fhir+json:
+              schema:
+                type: object
+                additionalProperties: { }
```

YAMLlint (1.37.1)

[error] too many spaces inside empty braces

(braces)